### PR TITLE
Improve actions on conversations

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

File: client/app/actions/conversation_action_creator.coffee [hunk 1]
```diff
@@ -34,11 +44,24 @@ module.exports =
                     label: t 'conversation undelete'
                     onClick: -> MessageActionCreator.undelete()
                 ]
-            LayoutActionCreator.notify t('conversation delete ok'), options
+            LayoutActionCreator.notify t('conversation delete ok', subject: messages[0].subject), options
```
Comment (frankrousseau) [inline client/app/actions/conversation_action_creator.coffee]: This line should be less than 80 chars long.

File: client/app/utils/translators/account_translator.coffee [hunk 2]
```diff
@@ -17,6 +17,7 @@ module.exports = AccountTranslator =
         weight2 = 400
         # Try to detect special mailboxes, first with flags, then by name
         # Already done on server, except for TEST accounts
+        raw.mailboxes = [] if not raw.mailboxes?
```
Comment (frankrousseau) [inline client/app/utils/translators/account_translator.coffee]: you can write it

```coffeescript
raw.mailboxes ?= []
````

**Comment by frankrousseau** [comment]
There are a lot of lines with a length > 80 chars. It makes things are to read.

**Comment by frankrousseau** [comment]
@aenario For this sprint could you first merge this one. We'll be a lot to participate. So, I think we should merge any PR quickly.

**Comment by aenario** [comment]
Done.

**Pull request merged**